src/System.Devices.Gpio/System.Devices.Gpio.Samples/LiquidCrystal.cs

```diff
@@ -0,0 +1,460 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.CompilerServices;
+
+namespace System.Devices.Gpio.Samples
+{
+    // Ported from Microsoft.IoT.Lightning Nuget package
```

Comment:
Please update the PR to include a THIRD-PARTY-NOTICES.TXT for the corefxlab root directory and add the License notice for Microsoft.IoT.Ligning to it.  See https://github.com/dotnet/corefx/blob/master/THIRD-PARTY-NOTICES.TXT as an example.

Please do not commit this PR, until I get a chance to review the PR again with this change included.